images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -75,12 +75,34 @@ Install-AndroidSDKPackages -AndroidSDKManagerPath $sdkManager `
                           -AndroidSDKRootPath $sdkRoot `
                           -AndroidPackages $androidToolset.additional_tools
 
+# NDKs
+$ndkLTSMajorVersion = $androidToolset.ndk.lts
+$ndkLatestMajorVersion = $androidToolset.ndk.latest
+
+$ndkLTSFullVersion = Get-AndroidPackagesByName -AndroidPackages $androidPackages `
+                -PrefixPackageName "ndk;" `
+                | Where-Object { $_ -Match "ndk;$ndkLTSMajorVersion.*" } `
```

Comment:
Can we get rid of the `| Where-Object ...` expression and pass the `$ndkLTSMajorVersion` as a package prefix?
```
-PrefixPackageName "ndk;$ndkLTSMajorVersion."
```

Reply:
Done.